src/index.js

```diff
@@ -1,5 +1,5 @@
 const server = require('./server');
 
-server().then(() => {
+server.server().then(() => {
```

Comment:
good catch, forgot about it, but I'm changing it a bit to:

```js
const { server } = require('./server');

server().then(() => {
  console.log('Server is running');
});
```

looks a bit better, right?

Reply:
Great!